### PR TITLE
force to validate min and max refresh interval to be positive

### PR DESCRIPTION
**Pull request by allencloud**
force to validate engine-refresh-min-interval and engine-refresh-max-interval to be positive

### CODE REVIEW

File: cli/manage.go [hunk 1]
```diff
@@ -244,8 +244,8 @@ func manage(c *cli.Context) {
 
 	refreshMinInterval := c.Duration("engine-refresh-min-interval")
 	refreshMaxInterval := c.Duration("engine-refresh-max-interval")
-	if refreshMinInterval == time.Duration(0)*time.Second {
-		log.Fatal("minimum refresh interval should be a positive number")
+	if refreshMinInterval <= time.Duration(0)*time.Second || refreshMaxInterval <= time.Duration(0)*time.Second {
```
Comment (jimmyxian) [inline cli/manage.go]: I think just using `refreshMinInterval <= time.Duration(0)*time.Second` here is OK

Comment (allencloud) [inline cli/manage.go]: Yes, you are right. Code in my way seems a little bit redundant.

**Comment by dongluochen** [comment]
LGTM

**Comment by jimmyxian** [comment]
LGTM

**Pull request merged**